test/utils/utils.go

```diff
@@ -75,6 +85,9 @@ func (p *PodmanTest) PodmanAsUserBase(args []string, uid, gid uint32, cwd string
 	}
 
 	runCmd := append(wrapper, podmanBinary)
+	if p.NetworkBackend == Netavark {
+		runCmd = append(runCmd, "--network-backend", "netavark")
+	}
```

Comment:
I think we should also force `--network-backend cni` since it will pick netavark when it is installed and no cni networks are present.